go/helper-image/delve-only-same-user.patch

```diff
@@ -4,14 +4,16 @@ the outside of the pod, and so connecting to the dlv port is refused.
 
 [1] https://github.com/go-delve/delve/pull/1764
 
---- cmd/dlv/cmds/commands.go
-+++ cmd/dlv/cmds/commands.go
-@@ -114,7 +114,7 @@ func New(docCall bool) *cobra.Command {
- 	rootCommand.PersistentFlags().StringVar(&buildFlags, "build-flags", buildFlagsDefault, "Build flags, to be passed to the compiler.")
- 	rootCommand.PersistentFlags().StringVar(&workingDir, "wd", ".", "Working directory for running the program.")
- 	rootCommand.PersistentFlags().BoolVarP(&checkGoVersion, "check-go-version", "", true, "Checks that the version of Go in use is compatible with Delve.")
+--- cmd/dlv/cmds/commands.go.orig	2021-09-23 00:05:04.000000000 -0400
++++ cmd/dlv/cmds/commands.go	2021-10-30 22:03:43.000000000 -0400
+@@ -130,8 +130,8 @@
+ 	rootCommand.PersistentFlags().StringVar(&initFile, "init", "", "Init file, executed by the terminal client.")
+ 	rootCommand.PersistentFlags().StringVar(&buildFlags, "build-flags", buildFlagsDefault, "Build flags, to be passed to the compiler. For example: --build-flags=\"-tags=integration -mod=vendor -cover -v\"")
+ 	rootCommand.PersistentFlags().StringVar(&workingDir, "wd", "", "Working directory for running the program.")
+-	rootCommand.PersistentFlags().BoolVarP(&checkGoVersion, "check-go-version", "", true, "Checks that the version of Go in use is compatible with Delve.")
 -	rootCommand.PersistentFlags().BoolVarP(&checkLocalConnUser, "only-same-user", "", true, "Only connections from the same user that started this instance of Delve are allowed to connect.")
++	rootCommand.PersistentFlags().BoolVarP(&checkGoVersion, "check-go-version", "", false, "Checks that the version of Go in use is compatible with Delve.")
 +	rootCommand.PersistentFlags().BoolVarP(&checkLocalConnUser, "only-same-user", "", false, "Only connections from the same user that started this instance of Delve are allowed to connect.")
  	rootCommand.PersistentFlags().StringVar(&backend, "backend", "default", `Backend selection (see 'dlv help backend').`)
-
- 	// 'attach' subcommand.
```

Comment:
nit: why remove this comment?

Reply:
Oh, this is a patch of a patch.  The previous patch was against Delve 1.6.0, whereas the new patch was against 1.7.2.

I'm going to try to get a patch accepted upstream to allow setting these values through `go build -ldflags -X` statements, so this patch can go away.